### PR TITLE
AndroidX updates 201909 (new navigation, paging, media2 and updates t…

### PR DESCRIPTION
**Pull request by moljac**
### Support Libraries Version (eg: 23.3.0):

AndroidX (new groups media2, navigation, paging and several new artifacts, plus numerous version bumps)

### Does this change any of the generated binding API's?

Some.

### Describe your contribution

new initial bindings and some updates

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -1,8 +0,0 @@
-<Project>
```
Comment (Redth) [inline Directory.Build.props]: Any reason this is gone?

Comment (moljac) [reply Directory.Build.props]: No idea. Most likely by accident.

Comment (moljac) [reply Directory.Build.props]: Let me find it and resurrect it.

**Pull request merged**